sounds/management/commands/csv_bulk_upload.py

```diff
@@ -116,14 +122,15 @@ def handle(self, *args, **options):
             user = User.objects.get(username=usernamef)
 
             # 1 create dir and move sound to dir
-            directory = self.user.profile.locations()['uploads_dir']
```

Comment:
wow this was a bug right? I mean the command does not really work (before applying this PR) right?

Reply:
I think so, it didn't work in my local.

Reply:
Oh I saw the comment in description now, nevermind.